xtract-core/src/main/scala/com/lucidchart/open/xtract/DefaultXmlReaders.scala

```diff
@@ -47,11 +52,9 @@ trait DefaultXmlReaders {
    */
   implicit val intReader: XmlReader[Int] = XmlReader { xml =>
     getNode(xml).flatMap { node =>
-      try {
-        ParseSuccess(node.text.toInt)
```

Comment:
I think this is a good thing to fix, but I'm not sure about the specific implementation. I think this would successfully parse something like "<a>abc123def</a>" as `123`, which I don't think is desirable. How about just changing to `ParseSuccess(node.text.trim.toInt)`?

Reply:
You are right. it's a bit over-engineered :)